api/orchestrator/orchestrator.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Fraunhofer AISEC
```

Comment:
As discussed, I would move this to a) a separate PR and b) wait for Go 1.18 generics